tests/flake8_json_reporter_test.py

```diff
@@ -1,19 +1,41 @@
+import json
 from argparse import Namespace
 
 import pytest
 from flake8.violation import Violation
 
+from flake8_json_reporter.reporters import DefaultJSON
 from flake8_json_reporter.reporters import FormattedJSON
 
 
 @pytest.fixture
-def formatter():
+def pretty_formatter():
```

Comment:
can you rebase out this part here -- it makes it a lot more difficult to see your change amongst this cosmetic change

Reply:
I don't feel it is a cosmetic change.

I added pretty_ to the fixture and the tests to differentiate between pretty-json formatting and default json formatting.

The original tests only covered pretty-json formatting and there were no test for other formatters.

If you do not like the name change, by second choice would be to move my tests to a new file.  But I leave that up to you.

Reply:
I agree with @asottile here. I missed it because I was on my phone